docs/development/contributing-to-docs.md

```diff
@@ -0,0 +1,139 @@
+
+If you need to update a Druid doc, locate and update the doc in the Druid repo, following the instructions below. Once a month, we run a script to update the Imply docs repo with recent updates to the Druid docs.
+
+## Druid repo branches
+
+The Druid team works on Apache Druid master, and then branches to 0.17, 0.18, etc and 017-iap (which stands for Imply Analytics Platform).
+
+See [CONTRIBUTING.md](https://github.com/apache/incubator-druid/blob/master/CONTRIBUTING.md) for instructions on contributing to Apache Druid.
+
+## Before you begin
+
+Before you can contribute to the Druid docs for the first time, you must complete the following steps:
+
+  1. Fork the [Druid repo](https://github.com/apache/druid). Your fork will be the ```origin``` remote.
+  2. Clone the Druid repo from your fork.
+  3. Set up your remotes locally ```upstream``` in the Druid repo in ```.git/config```:
+  ~~~~
+  [remote "upstream"]
+  	url = https://github.com/apache/druid.git
+  	fetch = +refs/heads/*:refs/remotes/upstream/*
+  	pushurl = no_push
+  [branch "master"]
+  	remote = upstream
+  	merge = refs/heads/master
+  [remote "origin"]
+  		url = https://github.com/{my-git-id}/druid.git
+  		fetch = +refs/heads/*:refs/remotes/upstream/*
+  [branch "master"]
+  		remote = origin
+  		merge = refs/heads/master
+  ~~~~
+
+  For ```upstream```, ```push_url = no_push``` means you won’t accidentally push to upstream.
+  Make sure to put your github id for {my-git-id}.
+  4. ```git config --list --show-origin``` to make sure you’ve got your email configured. If you need to set your email, you can set it per repo or globally. Global instructions [here](https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git).
+  5. Docusaurus?
+
+## Contributing
+
+  1. On branch ```master```, fetch the latest commit:
+
+  ~~~~
+  git fetch upstream
+
+  remote: Enumerating objects: 397, done.
+  remote: Counting objects: 100% (341/341), done.
+  remote: Compressing objects: 100% (181/181), done.
+  remote: Total 397 (delta 118), reused 255 (delta 98), pack-reused 56
+  Receiving objects: 100% (397/397), 266.19 KiB | 16.64 MiB/s, done.
+  Resolving deltas: 100% (118/118), completed with 44 local objects.
+  From https://github.com/apache/druid
+     819d706082..6c9f926e3e  0.21.0       -> upstream/0.21.0
+     8a3be6bccc..84aac4832d  master       -> upstream/master
+   * [new tag]               druid-0.21.0 -> druid-0.21.0
+
+  ➜ git reset --hard upstream/master
+  HEAD is now at 84aac4832d Add feature to automatically remove rules based on retention period (#11164)
+  ~~~~
+
+  Now you're up to date.
+
+  2. Create your working branch:
+  ~~~~
+  git checkout -b my-work
+  ~~~~
+  3. Make your changes, add, and commit:
+  ~~~~
+  git add my-change.md
+  git commit -m "i made some changes"
+  ~~~~
+  4. Test changes locally.
```

Comment:
In addition to Paul's recommendation on testing before commiting, this is probably what people should do to test.
```suggestion
  4. Test changes locally by building the site and navigating to your changes. In `website`, run `docusaurus-start`. By default, this starts the site on `localhost:3000`. 
```